scripts/synthetic/generate_pipeline_results_files.py

```diff
@@ -198,7 +198,7 @@ def generate_pipeline_results_files(
     # generate a cycle list so that generated data for each sample is predictable
     SYNTHETIC_DATA_POOL = cycle(SYNTHETIC_DATA)
 
-    df_metadata = load_data_from_csv(Path(metadata_file), EXPECTED_METADATA_HEADERS)
+    df_metadata = load_data_from_csv(Path(metadata_file), EXPECTED_METADATA_HEADERS["illumina"])
```

Comment:
Just to check, this is for the old synthetic pipeline right (which I assume only accepts Illumina samples)?

Also, can you not reference `ILLUMINA` here to avoid the magic string?

Reply:
I think synthetic is now totally unused, but I will fix this

Reply:
also I am assuming it only accepted illumina, I have no way of really knowing